src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -159,14 +159,11 @@
   <data name="PX1006Title" xml:space="preserve">
     <value>This order of view declarations will cause creation of one cache instance for the following DACs: {0} and {1}</value>
   </data>
-  <data name="PX1007Fix" xml:space="preserve">
-    <value>Create a DAC instance by using the factory method</value>
-  </data>
-  <data name="PX1007MessageFormat" xml:space="preserve">
-    <value>An instance of {0} must be created with the PXCache&lt;{0}&gt;.CreateInstance() factory method</value>
+  <data name="PX1007FixExcludeClass" xml:space="preserve">
+    <value>Exclude class from API Reference</value>
```

Comment:
@k-s-popova please review these 3 messages

Reply:
Exclude the class from the API Reference